plugins/modules/ipa_config.py

```diff
@@ -40,6 +40,12 @@
     aliases: ["primarygroup"]
     type: str
     version_added: '2.5.0'
+  ipagroupobjectclasses:
+    description: A list of group objectclasses.
+    aliases: ["groupobjectclasses"]
+    type: list
+    elements: str
+    version_added: '7.2.0'
```

Comment:
7.2.0 already has been released:
```suggestion
    version_added: '7.3.0'
```